Apps/Listener/dev-commands.lisp

```diff
@@ -162,6 +162,44 @@
           (format stream "~A bytes" object)
         (format stream "~,1F~A" (/ object (expt 1000 idx)) (nth idx suffixes))))))
 
+;;; In a textual-view lists and vectors are presented as
+;;; comma-separated items, rather than in a lisp-readable format.  For
+;;; the Listener it makes better sense for these to look like lisp
+;;; forms when they appear in menus.
+(define-presentation-method present ((object list) (type sequence)
+                                     stream
```

Comment:
I don't yet fully grok all the stream dances that happen with CLIM, so that's possible but I'd have to do a lot more staring to figure that out.

No other part of McCLIM or any of the demonstration apps specialize presentation types for the `textual-menu-view`. I'm not sure what we'd be stomping on my accident for the Listener to specialize them.